blockchain/vm/evm.go

```diff
@@ -572,8 +572,6 @@ func (evm *EVM) GetPrecompiledContractMap(addr common.Address) map[common.Addres
 	case evm.chainRules.IsIstanbul:
 		return PrecompiledContractsIstanbulCompatible
 	default:
-		// It should not happen.
-		logger.Error("VmVersion is not zero, but it uses PrecompiledContractsByzantiumCompatible")
```

Comment:
I couldn't see when an EOA addresss can be a param of this method `GetPrecompiledContractMap`. Is there any case?

Reply:
@jeongkyun-oh EOA address can be inserted as an `addr` when caller is `isProgramAccount`. 
https://github.com/klaytn/klaytn/blob/d7a817488a2aa86a6343e127e1f970a460ebd9c9/blockchain/vm/evm.go#L55-L61